cabal.project

```diff
@@ -1,3 +1,13 @@
 packages: streamly-examples.cabal
 
+source-repository-package
+    type:       git
+    location:   https://github.com/composewell/streamly
+    tag:        f1a04362162d707054115a34563e1c35199bb796
+
+source-repository-package
+    type:       git
+    location:   https://github.com/composewell/unicode-data
+    tag:        e3208a437b4b71613656c0aff9165442256f89ed
```

Comment:
We can add the released package to default.nix, instead of adding this here.

Reply:
Like this: https://github.com/composewell/streamly-examples/commit/0926cd29365671ce91c1dd3bce21eb52b320c194